### PR TITLE
{compiler} [intel-2015a] LLVM 3.7.0 (REVIEW)

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/l/LLVM/LLVM-3.6.0.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+description = """The LLVM Core libraries provide a modern source- and target-independent
+ optimizer, along with code generation support for many popular CPUs
+ (as well as some less common ones!) These libraries are built around a well
+ specified code representation known as the LLVM intermediate representation
+ ("LLVM IR"). The LLVM Core libraries are well documented, and it is
+ particularly easy to invent your own language (or port an existing compiler)
+ to use LLVM as an optimizer and code generator."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+# Do not set optarch to True: it will cause the build to fail
```
Comment (boegel) [inline easybuild/easyconfigs/l/LLVM/LLVM-3.6.0.eb]: any more details? what kind of issue would you run into (and any idea why exactly)?

File: easybuild/easyconfigs/l/LLVM/LLVM-3.6.0.eb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+toolchainopts = {'optarch': False}
+
+source_urls = ["http://llvm.org/releases/%(version)s"]
+sources = ["llvm-%(version)s.src.tar.xz"]
+
+builddependencies = [
+    ('CMake', '3.1.3'),
```
Comment (boegel) [inline easybuild/easyconfigs/l/LLVM/LLVM-3.6.0.eb]: use latest 3.2.2?

File: easybuild/easyconfigs/l/LLVM/LLVM-3.6.0.eb (hunk 1 above)
Comment (hajgato) [inline easybuild/easyconfigs/l/LLVM/LLVM-3.6.0.eb]: No Idea. I just copied form CLang eb. O will check, though

Comment (boegel) [inline easybuild/easyconfigs/l/LLVM/LLVM-3.6.0.eb]: ok, maybe @wpoely86 knows?

Comment (hajgato) [inline easybuild/easyconfigs/l/LLVM/LLVM-3.6.0.eb]: BTW is WIP :)

Comment (boegel) [inline easybuild/easyconfigs/l/LLVM/LLVM-3.6.0.eb]: I know, just wanted to give it an early look ;)

Comment (wpoely86) [inline easybuild/easyconfigs/l/LLVM/LLVM-3.6.0.eb]: Older versions of Clang failed with `-O3`. Haven't checked it on the recent (3.6). I should recheck.

**Comment by wpoely86** [comment]
@hajgato what do you want/need with this easyconfig? Normally, the LLVM is also part of Clang and installed under it. Or are some thing missing?

**Comment by hajgato** [comment]
@wpoely86 I am building Mesa with gallium-llvm driver. It needs llvm, but not the whole CLang as builddep. I thought that I will not use the whole CLang.

**Comment by hajgato** [comment]
@wpoely86 Anyway, I am open to everything.

**Comment by wpoely86** [comment]
@hajgato that drivers is slooooooooooooooooow. I would avoid using it.

**Comment by hajgato** [comment]
@wpoley86 Anything better? I got the info from here: http://www.paraview.org/Wiki/ParaView_And_Mesa_3D

**Comment by wpoely86** [comment]
@hajgato yes, not using egl/glx related stuff on the HPC :wink:

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by boegel** [comment]
@hajgato: please review/merge https://github.com/hajgato/easybuild-easyconfigs/pull/24

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/c93aae58c900213ab0a1 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/b3806be950e41f8df3f2 for a full test report.

**Comment by fgeorgatos** [comment]
lgtm!

**Comment by boegel** [comment]
going in, thanks @hajgato!

**Pull request merged**